cs/azure/Trainer/Operations/EvalOperation.cs

```diff
@@ -138,17 +139,23 @@ private IEnumerable<EvalData> OfflineEvaluateInternal(TrainerResult trainerResul
             };
 
             for (int action = 1; action <= trainerResult.ProgressivePrediction.Length; action++)
+            {
+                string tag;
+                if (!trainerResult.ActionsTags.TryGetValue(action, out tag))
+                    tag = action.ToString(CultureInfo.InvariantCulture);
+
                 yield return new EvalData
                 {
-                    PolicyName = $"Constant Policy {action}",
+                    PolicyName = $"Constant Policy {tag}",
                     JSON = JsonConvert.SerializeObject(
                     new
                     {
-                        name = $"Constant Policy {action}",
+                        name = $"Constant Policy {tag}",
```

Comment:
Just wanted to reuse the string that was set to PolicyName here, so something like:

PolicyName = $". . .",
. . .
new
{
    name = PolicyName,
. . .